docs/ops/arithmetic/Add_1.md

```diff
@@ -12,29 +12,29 @@
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
+    * *none* - no auto-broadcasting is allowed, all input shapes must match
     * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
```

Comment:
@lazarevevgeny do we have broadcasting rules described somewhere in our spec? I think this reference to ONNX docs should be avoided and since these rules are common to numerous ops we should also have one place with description which we could refer to.

Reply:
The most detailed one is in the `docs/ops/movement/Broadcast_1.md`.

Reply:
`docs/ops/movement/Broadcast_1.md` also refers to ONNX docs, additionally in broadcast description there is a link to `https://docs.scipy.org` for numpy broadcasting details.

Reply:
My view is that our spec should be standalone as much as possible (links to papers, and articles explaining operation in details are fine). I know that the practice before was to refer to other frameworks, but for me it is unnecessary and unmanaged dependency (what if link will go away? or it will be changed significantly?) and most of all makes it harder for the reader to consume the spec. 

This is my view I can't enforce it on everyone alone and it not makes sense to do it partially, hence I've involved @lazarevevgeny in this discussion.

Reply:
@lazarevevgeny, may you help to decide how to progress with this issue?

Reply:
Sorry for a delay. Lets have one place (probably the spec for Broadcast op is the best place) where we describe our broadcasting modes in details with good examples. Then we can update all specs which currently refer to the numpy spec to point to this spec.

Reply:
ONNX link is removed and now it redirects to local Broadcast op.